recipes/imgui-bundle/meta.yaml

```diff
@@ -0,0 +1,125 @@
+{% set version = '1.6.0' %}
+
+package:
+  name: imgui-bundle
+  version: {{ version }}
+
+source:
+  # url: https://github.com/pthom/imgui_bundle/archive/refs/tags/v{{ version }}.tar.gz
+  # sha256: ffae861ff9fd47c95cc56807f028d5fe78a700f92a2a43c43f31ea2358542a68
+  git_url: https://github.com/pthom/imgui_bundle.git
+  git_tag: v{{ version }}
+  entry_points:
+    - demo_imgui_bundle = imgui_bundle.demos_python.demo_imgui_bundle:main
+    - imgui_bundle_demo = imgui_bundle.demos_python.demo_imgui_bundle:main
```

Comment:
@pthom i just added these 3 lines and this should resolve the issues with the windows demo. gotta wait 3 hours for the new builds though.